sockeye/utils.py

```diff
@@ -317,28 +392,18 @@ def acquire_gpus(num_gpus:int, lock_dir: str = "/tmp", retry_wait_min: int = 30,
 
 
 @contextmanager
-def _acquire_single_gpu(lock_dir: str = "/var/lock"):
+def _acquire_single_gpu(candidates: List[int], lock_dir: str = "/var/lock"):
```

Comment:
you could make the lock_dir argument not optional or also change it to /tmp to be consistent